src/Context.cpp

```diff
@@ -283,6 +283,7 @@ std::string configurationDefaults =
   "#sync.server.encryption_secret                 # Encryption secret for sync to a server\n"
   "#sync.server.origin                            # Origin of the sync server\n"
   "#sync.local.server_dir                         # Directory for local sync\n"
+  "#sync.gcp.credentialpath                       # Path to JSON file containing credentials to authenticate GCP Sync\n"
```

Comment:
The configuration parameters have `_` between words, so this should be `credential_path`.

Reply:
Okay!

Reply:
Okay! 
I have changed parameters throughout, just in case